devops/resources/manage-newrelic-server-alerts.sh

```diff
@@ -0,0 +1,5 @@
+cd sysadmin/newrelic
+
+pip install -r requirements.txt
+
+python add_app_servers_to_server_policy.py --new-relic-api-key ${NEW_RELIC_API_KEY} --alert-policy-name ${ALERT_POLICY_NAME} ${APP_NAMES}
```

Comment:
Hannah and I fixed this at the system level, but just confirm that the api key gets *****ed in the jenkins console